src/components/TodoItem.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './TodoItem.module.css';
+
+class TodoItem extends React.Component {
+  state = {
+    editing: false,
+  };
+
+  componentWillUnmount() {
+    console.log('Cleaning up...');
```

Comment:
- [ ] Kindly remove all `console.logs()`'s from the project. (It's a good `ES6` practice)